.env.rc0

```diff
@@ -0,0 +1,150 @@
+# Don't use "//" for comments in this file.
+# This file is meant to be executed as a bash script for testing.
+ENV_TYPE="production"
+
+GETH_VERBOSITY=2
+
+VM_BASED=true
```

Comment:
Given that we are running this internally, I think it'd be cheaper ($$ wise) for us to not have this be VM_BASED, wdyt?

Reply:
Agreed this is probably not necessary